crates/sui-indexer-alt/src/lib.rs

```diff
@@ -107,13 +109,27 @@ pub async fn setup_indexer(
 
     let retry_interval = ingestion.retry_interval();
 
+    // Prepare the store for the indexer
+    let store = Db::for_write(database_url, db_args)
+        .await
+        .context("Failed to connect to database")?;
+
+    // we want to merge &MIGRATIONS with the migrations from the store
+    store
+        .run_migrations(postgres::migrations(Some(&MIGRATIONS)))
```

Comment:
Why not make `run_migrations on `Db` automatically add the watermark migrations?